cmd/runtimetest/main.go

```diff
@@ -191,6 +191,20 @@ func validateSysctls(spec *rspec.Spec) error {
 	return nil
 }
 
+func validateROPaths(spec, *rspec.Spec) error {
+	fmt.Println("validating Readonly Paths")
+	for _, v := range spec.Linux.ReadonlyPaths {
+		fi, err := os.Stat(v)
```

Comment:
@wking  Yes, but `runtimetest` as configured process in config.json, it should be executed in container namespace. So, I think there is no problem.